src/main/scala/com/stratio/khermes/helpers/faker/generators/TicketsCarrefourGenerator.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.khermes.helpers.faker.generators
+
+import com.stratio.khermes.commons.constants.AppConstants
+import com.stratio.khermes.commons.exceptions.KhermesException
+import com.stratio.khermes.commons.implicits.AppSerializer
+import com.stratio.khermes.helpers.faker.FakerGenerator
+import com.typesafe.scalalogging.LazyLogging
+import org.json4s.native.Serialization._
+
+import scala.util.{Failure, Success, Try}
+
+case class TicketsCarrefourGenerator(locale: String) extends FakerGenerator
```

Comment:
In my opinion, the generators should be generic and shouldn't be related to any customer. It would be nice a generic ticket generator by mixing existing custom generators within a twirl template and/or creating some new custom generators when you need.